code/__HELPERS/pronouns.dm

```diff
@@ -386,3 +386,33 @@
 		temp_gender = gender
 	if(temp_gender != PLURAL)
 		return "es"
+
+/datum/mind/p_they(temp_gender)
```

Comment:
there has to be a better way to handle this than like 15 procs

Reply:
This is how we typically do pronouns, largely

Reply:
This is generally how BYOND handles them, yeah. It's clunky but mostly sane because there's only ever going to be four options - it, they, he, or she.